### PR TITLE
Migrate to CocoaPods 1.0.1

### PR DESCRIPTION
**Pull request by corinnekrych**
@jcesarmobile mind a review?

### CODE REVIEW

**Comment by jcesarmobile** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ A small and handy library written in [Swift](https://developer.apple.com/swift/)
 
 2. Get the dependencies
 
-The project uses [OHHTTPStubs](https://github.com/AliSoftware/OHHTTPStubs) framework for stubbing its http network requests and utilizes [cocoapods](http://cocoapods.org) 0.36+ release for handling its dependencies. As a pre-requisite, install [cocoapods](https://cocoapods.org/) and then install the pod. On the root directory of the project run:
+The project uses [OHHTTPStubs](https://github.com/AliSoftware/OHHTTPStubs) framework for stubbing its http network requests and utilizes [cocoapods](http://cocoapods.org) 0.36+ release for handling its dependencies. As a pre-requisite, install [CocoaPods](https://cocoapods.org/) and then install the pod. On the root directory of the project run:
 ```bash
```
Comment (matzew) [inline README.md]: 0.36+ ?

**Comment by danielpassos** [comment]
👍

**Comment by jcesarmobile** [comment]
LGTM, waiting for travis

**Comment by corinnekrych** [comment]
Clueless why travis is failing, running
xctool -workspace AeroGearPush.xcworkspace -scheme AeroGearPush -sdk iphonesimulator build test
with xctool version 0.2.9 and xcodebuilt version Xcode 7.3.1 (Build version 7D1014) - all similar to travis gives me a green build...
#puzzled why a timeout????

**Pull request merged**